rskj-core/src/main/java/co/rsk/peg/BridgeEvents.java

```diff
@@ -80,6 +80,12 @@
                     new CallTransaction.Param(false, Fields.AMOUNT, SolidityType.getType(SolidityType.UINT256)),
                     new CallTransaction.Param(false, Fields.REASON, SolidityType.getType(SolidityType.INT256))
             }
+    ),
+    BATCH_PEGOUT_CREATED("batch_pegout_created",
+        new CallTransaction.Param[]{
+            new CallTransaction.Param(true, Fields.BTC_TX_HASH, SolidityType.getType(SolidityType.BYTES32)),
+            new CallTransaction.Param(false, Fields.RELEASE_RSK_TX_HASH, SolidityType.getType(SolidityType.BYTES32))
```

Comment:
```suggestion
            new CallTransaction.Param(false, Fields.RELEASE_RSK_TX_HASHES, SolidityType.getType(SolidityType.BYTES))
```
It's a list of all the rsk tx hashes included in the batch

Reply:
Resolved